scripts/setup-vm-user.sh

```diff
@@ -17,10 +17,12 @@ usermod -a -G adm,cdrom,sudo,dip,plugdev,lpadmin,sambashare $LOGIN_USER
 # ensure the new user can do passwordless sudo
 echo "$LOGIN_USER ALL=(ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/$LOGIN_USER
 
-# set the new user as the default in the login screen
+# set the new user as the default in the login screen and end the current (vagrant) user's gnome session
 mkdir -p /etc/gdm3
 > /etc/gdm3/custom.conf
 echo "[daemon]" >> /etc/gdm3/custom.conf
 echo "# Enabling automatic login" >> /etc/gdm3/custom.conf
 echo "AutomaticLoginEnable = true" >> /etc/gdm3/custom.conf
 echo "AutomaticLogin = $LOGIN_USER" >> /etc/gdm3/custom.conf
+
+systemctl restart display-manager
```

Comment:
so this did the trick to logout the user?

Reply:
It restarts the display server, so it will log out the current user (vagrant) and (because of the script above) immediately log in the newly created user.

Reply:
Thanks!